src/models/User.js

```diff
@@ -1,10 +1,33 @@
 import mongoose from 'mongoose';
 const userSchema = new mongoose.Schema({
-    username: {
+  firstName: {
       type: String,
       required: true,
       unique: true
     },
+    lastName: {
+      type: String,
+      required: true,
+      unique: true
+    },
+    countryOfOrigin: {
+      type: String,
+      required: true,
+    },
+    countryCode: {
+      type: String,
+      required: true,
+    },
+    photo: {
+      type: String,
```

Comment:
What is photo? base64 string?

Reply:
It's just a link to an image online it's not a real feature